docs/source/tutorials/configuration.rst

```diff
@@ -130,7 +129,7 @@ open the file after it is rendered.
 The config files
 ****************
 
-As the last example shows, executing manim from the command-line may involve
+As the last example shows, executing manim from the command-line might involve
 using many flags at the same time.  This may become a nuisance if you must
```

Comment:
For consistency, line 133 should also use might. Otherwise, this change should be reverted so that the overall paragraph is consistent in the usage of 'may' and 'might'.

Reply:
> For consistency, line 133 should also use might. Otherwise, this change should be reverted so that the overall paragraph is consistent in the usage of 'may' and 'might'.

sure, I think I will revert this bit